dev-python/axolotl/axolotl-0.1.42.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_P="python-${P}"
+MY_PN="python-${PN}"
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="A python port of libsignal-protocol-java"
+HOMEPAGE="https://github.com/tgalal/python-axolotl"
+SRC_URI="https://github.com/tgalal/${MY_PN}/archive/${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/axolotl-curve25519[${PYTHON_USEDEP}]
+	dev-python/protobuf-python[${PYTHON_USEDEP}]
+	dev-python/pycrypto[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]"
+
+DEPEND="${DEPEND}"
+
+S="${WORKDIR}/${MY_P}"
```

Comment:
Tests!

Reply:
Added and they pass locally for me fine.